src/components/ConnectToWallet/ConnectToWallet.tsx

```diff
@@ -81,8 +82,13 @@ const ConnectToWallet: React.FC<ConnectToWalletProps> = ({ onConnect }) => {
 		}
 	};
 
+	//here needs to be stored the state or relay of the disconnect on the localStorage, 
+	//then in the mount or constructor this needs to chef if its connected, 
+	//if its connected, then disconnect at start
+
 	const disconnect = () => {
 		deactivate();
+		localStorage.setItem("disconnected","true") //set the state to disconnected
```

Comment:
Please make sure to run your code through the code styler / prettier. There should be spaces after commas.